projects/plugins/jetpack/to-test.md

```diff
@@ -1,40 +1,31 @@
-- Navigate back to `/wp-admin/admin.php?page=jetpack#/recommendations/site-type` and select both "I build or manage this site for a client" and "This is an e-commerce site" and click on Continue.
-- You should see the Agencies recommendation first, select "Not now".
-- Next you should see the WooCommerce recommendation.
-- That is all for testing these updated recommendations.
-
-**And to test the mobile app links:**
-
-- Navigate to `/wp-admin/admin.php?page=jetpack#/recommendations/summary`
-- On a desktop or laptop computer, a QR code for the Jetpack mobile app will be displayed next to the `jetpack.com/mobile` URL.
-- When visiting the same recommendations page on a mobile device, the respective app store button should be shown for well known devices and mobile operating systems.
-- If you don't have a mobile device to test with, you might try using Chrome to simulate the user agent:
-  - Open the Chrome Dev Tools
-  - Click the vertical menu option > More tools > Network conditions
-  - In the Network conditions tab, uncheck the "Use browser default" for the user agent, and select an iOS or Android one for testing.
-  - When refreshing the recommendations page you should see the expected mobile app store button.
+### Jetpack Forms block
+
+Forms block got several updates and bug fixes in this version. To test, try the following:
+
+- Make sure the testing site has Jetpack connected to your account.
+- Create a post and add a Form block to it. 
+- Hover your pointer between two Form child blocks (eg. Name and Email in Contact form). The Block Inserter (plus sign with Add block title) should appear, allowing you to easily insert new fields between the blocks.
+- Check the Dimensions section in the sidebar and make sure it already has padding setting (16 px). Try editing padding.
+- Try editing Field widths for different fields and make sure that what you see in the editor matches the frontend.
+- Add a Contact form and select the Form block container. Make sure the edit button doesn't show up (pen icon). All form settings such as email address and subject line should be available in the sidebar.
+- Lose and regain focus on the Form block by clicking inside of it. Clicking the empty space between child elements should select the main Form block.
+- See if form related blocks show up in the block library when trying to insert a new child Form block. Try setting the focus on a child block and adding a block. The section with the Form blocks should have a title "FORMS".
+- Add a Contact Form and add a Consent block to it (any type). Configure the Email address to send to to your email. Pick "Show summary of submitted fields" as On submit action. Submit the form on the frontend and see that the "Consent: Yes" is visible on the Success page.
+
+### Jetpack Social
+
+Jetpack Social connections are no longer counted in the Subscriptions block. To test, try the following:
```

Comment:
> Jetpack Social connections are no longer counted in the Subscriptions block

To the average person, I think this could be a tough statement to decipher. We should be very clear about what is changing here. Also reference internal: p58i-dod-p2